script.py

```diff
@@ -1,73 +1,67 @@
-import time 
-from selenium import webdriver 
+import time
+from selenium import webdriver
 from selenium.webdriver.support.select import Select
 
-driver = webdriver.Chrome()
+driver = webdriver.Chrome('D:/webdriver/chromedriver.exe')
 url = "https://gu.icloudems.com/corecampus/index.php"
-  
-# Opening the website 
-driver.get(url) 
+
+# Opening the website
+driver.get(url)
 driver.maximize_window()
 
 time.sleep(2)
 username = driver.find_element_by_xpath('//*[@id="useriid"]')
 password = driver.find_element_by_xpath('//*[@id="actlpass"]')
 
-
-#Change user name and passwords with your icloud username and password
-username.send_keys('username')  ##################################### Change This Text ############################
-password.send_keys('password')  ##################################### Change This Text ############################
+# Change user name and passwords with your icloud username and password
+username.send_keys('17SCSE112003')  ##################################### Change This Text ############################
+password.send_keys('Gourav8211@')  ##################################### Change This Text ############################
 
 time.sleep(1)
 
-login  = driver.find_element_by_xpath('//*[@id="psslogin"]')
+login = driver.find_element_by_xpath('//*[@id="psslogin"]')
 login.click()
 
-
 time.sleep(2)
 
 feedback = driver.find_element_by_xpath('/html/body/div[1]/div/div/div[4]/div/div/div[7]/a/img')
 feedback.click()
 
-
 time.sleep(2)
 
-
 selectClass = driver.find_element_by_xpath('//*[@id="classid"]')
 sel = Select(selectClass)
-#change this text
-sel.select_by_visible_text('B.Tech in CSBA 2017-18 IV Sem VII')  ##################################### Change This Text ############################
 
+sel.select_by_index(1)
+
+time.sleep(10)
+#search = driver.find_element_by_class_name('fa-search')
+# search = driver.find_element_by_xpath('/html/body/div[1]/div/div[1]/div[3]/div/div[2]/div/form/div/div[3]/button')
+#search.click()
 
-time.sleep(1)
-search = driver.find_element_by_class_name('fa-search')
-search.click()
 
-time.sleep(2)
 
 start = driver.find_element_by_class_name('btn-rounded')
+#start = driver.find_element_by_xpath('/html/body/div[1]/div[1]/div/div[3]/div[2]/div[2]/center/input')
 start.click()
 
-time.sleep(4)
-
+time.sleep(2)
```

Comment:
Increase time to 5 seconds. This page take more time to load on slow internt

Reply:
I have done all the changes thanks for pointing them out.